.gitignore

```diff
@@ -0,0 +1,2 @@
+# Spack
+__pycache__/
```

Comment:
Not sure if intentionally included here?

Reply:
Rookie mistake! Worth adding it to the repository `.gitignore` (I feel it is something that could happen if someone uses `alps-cluster-config` directly as a repository in their Spack configurations, or is it one of those things that should just go on my own global ignore?

Reply:
You already saw this: https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file. I don't mind either way, so I'll leave it to someone else to propose removing it :wink: